### ISSUE TITLE
VSH nodes say "device limit reached" for most of my virtual devices

### ISSUE THREAD

**Issue by epower53** [csuermann/node-red-contrib-virtual-smart-home#151]
It appears that I may have some stale devices in my config, but I can't see them in Alexa or in the VSH node config. Most of my virtual device nodes are non-functional at present, showing "device limit reached." I have a total of 25 of these configured, and only 7 are currently working and talking to Alexa. First 6 digits of my thing ID: b19db8 
![image](https://user-images.githubusercontent.com/72056686/191523771-7eb8d97d-3b55-4553-9dea-47176b411f01.png)

![image](https://user-images.githubusercontent.com/72056686/191520031-def69df4-511e-4e73-b5d3-889f2f06f21e.png)

**Comment by epower53** [comment]
Looks like maybe the issue is due to this change:

https://github.com/csuermann/virtual-smart-home/commit/6a9fb04a36337369eb2f09bce84a2448379e8b53

**Comment by epower53** [comment]
Upgraded my package to the latest release, and I can now see that you are, indeed, moving to a subscription model. Was a warning about this change posted somewhere? It sure surprised me...

**Comment by epower53** [comment]
Went to the subscription payment site and stopped cold. Your merchant processing vendor doesn't have Paypal as an option. I've never heard of "Stripe" and I'm not about to give them my credit card. I strongly urge you to find a way to accept Paypal as a payment method. Will continue with my hobbled 7-device free account for now until you make the change, or until I find another option for Node-red <-> Alexa.

**Comment by csuermann** [comment]
Hi @epower53,

I'm sorry this change caught you by surprise! Unfortunately I couldn't announce the introduction of the "Pro" plan apart from here (#148), as I don't have a business relationship with the users of my VSH skill that would justify sending out mass emails to them. 

I chose Stripe as a payment processor as I found their product better suited for managing a subscription business than PayPal. Interesting that you've not heard of them yet! They have close to 10k employees and power almost every other e-commerce checkout flow out there...
